test/internal/kubernetes/utils.go

```diff
@@ -431,3 +449,19 @@ func HasWindowsNodes(ctx context.Context, clientset *kubernetes.Clientset) (bool
 	}
 	return false, nil
 }
+
+func MustRestartDaemonset(ctx context.Context, clientset *kubernetes.Clientset, namespace, daemonsetName string) error {
```

Comment:
Does this not have a happy path? Looks like every return is resulting in an error.

Reply:
It does, the last `errors.Wrap()` will return nil, if the err coming back 
`Wrapf returns an error annotating err with a stack trace at the point Wrapf is called, and the format specifier. If err is nil, Wrapf returns nil.`